src/components/graph/menus/modification-list-item.js

```diff
@@ -109,22 +116,52 @@ export const ModificationListItem = ({
         [modif, handleToggle]
     );
 
+    const getBranchStatusModificationValues = (modification) => {
+        return {
+            action: modification?.action,
+            energizedEnd: modification?.energizedVoltageLevelId,
+            computedLabel: <strong>{modification?.equipmentId}</strong>,
+        };
+    };
+
+    const getEquipmentAttributeModificationValues = (modification) => {
+        return {
+            equipmentAttributeName: modification?.equipmentAttributeName,
+            equipmentAttributeValue: modification?.equipmentAttributeValue,
+            computedLabel: <strong>{modification?.equipmentId}</strong>,
+        };
+    };
+
     useEffect(() => {
         if (!studyUuid || !currentNode || !modif) {
             return;
         }
-        setComputedValues({
-            energizedEnd: modif.energizedVoltageLevelId,
-            computedLabel: <strong>{getComputedLabel()}</strong>,
-        });
+        const modificationValues = JSON.parse(modif.messageValues);
+
+        switch (modif.messageType) {
+            case MODIFICATION_TYPES.BRANCH_STATUS_MODIFICATION.type:
```

Comment:
Rename all `computedValues` by 'computedLabelValues'
Also the state and `setComputedValues`